public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.ts

```diff
@@ -85,6 +85,7 @@ export interface LoadDashboardOptions {
   slug?: string;
   type?: string;
   urlFolderUid?: string;
+  targetVersion?: 'v1' | 'v2';
```

Comment:
This is another place where i think we are unnecessarily mixing concern. `LoadDashboardOptions` is transporting info about input, not the output. We should separate these models.